R/financial.R

```diff
@@ -17,14 +17,30 @@
 dm_financial <- function() {
   stopifnot(rlang::is_installed("RMariaDB"))
 
-  my_db <- DBI::dbConnect(
-    RMariaDB::MariaDB(),
-    username = "guest",
-    password = "relational",
-    dbname = "Financial_ijs",
-    host = "relational.fit.cvut.cz"
+  my_db <- try(
+    DBI::dbConnect(
+      RMariaDB::MariaDB(),
+      username = "guest",
+      password = "relational",
+      dbname = "Financial_ijs",
+      host = "relational.fit.cvut.cz"
+    ),
+    silent = TRUE
   )
 
+  if (inherits(my_db, "try-error")) {
+    my_db <- try(
+      DBI::dbConnect(
+        RMariaDB::MariaDB(),
+        username = "student",
+        password = "tx5mvyRQqD",
+        dbname = "loan_application",
+        host = "db-edu.pacha.dev"
+      ),
+      silent = TRUE
+    )
+  }
```

Comment:
Can you extract a function here, please? The function should have an argument `source = "relational.fit"`, the other option would be `source = "db-edu"` .